wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -116,6 +119,29 @@ public void testGetEnabledJobsByType() throws IOException {
     Files.deleteIfExists(path2);
   }
 
+  @Test
+  public void testMultipleLinesSameProperties() throws IOException, ConfigurationException {
+    String dir = "/tmp/";
+    String propertyStr =
+        "var1=foo\n" + "var1=foo\n" + "var1=foo\n" + "var2=foo\n" + "var2=foo\n" + "var3=foo\n" + "var1=foo\n"
+            + "var4=foo\n";
+    Properties props = new Properties();
+    Path path = createPropertiesFile(propertyStr);
+
+    for (File file : new File(dir).listFiles()) {
```

Comment:
Why? Don't you already have the path from the previous line?

Reply:
Updated it now.